src/App.tsx

```diff
@@ -1,99 +1,13 @@
 /* eslint-disable max-len */
```

Comment:
Remove the 'eslint-disable max-len' comment if it's not required. It's better to follow the ESLint rules for better code quality.